hermes-management/src/main/java/pl/allegro/tech/hermes/management/infrastructure/kafka/service/retransmit/PartitionOffsetChange.java

```diff
@@ -0,0 +1,23 @@
+package pl.allegro.tech.hermes.management.infrastructure.kafka.service.retransmit;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class PartitionOffsetChange {
```

Comment:
good observation. I have removed PartitionOffsetChange class. However I also added @JsonCreator annotation to PatitionOffset class due to deserialization done in integration test for retransmission.